index.d.ts

```diff
@@ -19,7 +19,7 @@ declare namespace JwksRsa {
   }
 
   interface ClientOptions {
-    jwksUri: string;
```

Comment:
You need to leave `jwksUri` in or it will be a breaking change, just mark it as `@deprecated`

Reply:
Since `jwks` and `jwksUri` are mutually exclusive, you'll need to do something like:

```typescript
 class JwksClient {
    constructor(options: ClientOptions | DeprecatedClientOptions);
   ...
  }

interface ClientOptions {
  jwks: string | object[];
  rateLimit?: boolean;
  cache?: boolean;
  cacheMaxEntries?: number;
  cacheMaxAge?: number;
  jwksRequestsPerMinute?: number;
  proxy?: string;
  strictSsl?: boolean;
  requestHeaders?: Headers;
  timeout?: number;
}

interface DeprecatedClientOptions extends Omit<ClientOptions, 'jwks'> {
  jwksUri: string;
}
```